data-node/src/main/java/org/graylog/datanode/initializers/JwtTokenValidator.java

```diff
@@ -43,28 +46,18 @@ public JwtTokenValidator(@Named("password_secret") final String signingKey) {
     public void verifyToken(String token) throws TokenVerificationException {
         SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.HS256;
         Key signingKey = new SecretKeySpec(this.signingKey.getBytes(StandardCharsets.UTF_8), signatureAlgorithm.getJcaName());
-        final JwtParser parser = Jwts.parserBuilder()
+        final JwtParser parser = Jwts.parser()
                 .setSigningKey(signingKey)
                 .requireSubject(REQUIRED_SUBJECT)
                 .requireIssuer(REQUIRED_ISSUER)
                 .build();
         try {
-            final Jwt parsed = parser.parse(token);
-            verifySignature(parsed, signatureAlgorithm);
+            parser.parse(token);
+        } catch (UnsupportedJwtException e) {
+            LOG.warn("Token format/configuration is not supported", e);
+            throw new TokenVerificationException("Token format/configuration is not supported");
```

Comment:
Why are we now logging the error here and throwing the exception without a cause parameter?

Reply:
This is a replacement for the original NONE algorithm handling, which has been also just throwing exceptions without any cause. The message is then directly propagated as an API response, so we don't want to provide too much details. Additionally, we now have an actual cause, so we may also log it, to not swallow it completely.

Reply:
Unified the handling, propagate the cause now and removed logging